security/tpm/src/tpm.ts

```diff
@@ -3,22 +3,428 @@
 
 'use strict';
 import { errors } from 'azure-iot-common';
+import * as machina from 'machina';
+import * as tss from 'tss.js';
+import { Tpm, TPM_HANDLE, TPM_ALG_ID, TPM_RC, TPM_PT, TPMA_OBJECT, TPMT_PUBLIC, TPM2B_PRIVATE } from 'tss.js';
+import * as crypto from 'crypto';
+import base32Encode = require('base32-encode');
+
+import * as dbg from 'debug';
+
+const debug = dbg('azure-iot-security-tpm:tpmclient');
```

Comment:
>tpmclient [](start = 42, length = 9)

nit: TpmSecurityClient (I'm trying to enforce the package-name:ClassName convention)

Reply:
Fixed.

---
In reply to: [157881425](https://github.com/Azure/azure-iot-sdk-node/pull/185#discussion_r157881425) [](ancestors = 157881425)